Scripts/script-Base64Encode.yml

```diff
@@ -0,0 +1,26 @@
+commonfields:
+  id: Base64Encode
+  version: -1
+name: Base64Encode
+script: |-
+  var result = Base64.encode(args.input);
```

Comment:
What's Base64 - are we exposing this somewhere?
![screen shot 2017-04-21 at 2 02 37 pm](https://cloud.githubusercontent.com/assets/1479247/25296051/338766c0-269b-11e7-8b7f-219744362f98.png)